src/components/AssetCheckoutScanner.tsx

```
@@ -0,0 +1,80 @@
+
+        <div className={`p-4 rounded-lg flex items-center gap-4 ${scannedUser ? 'bg-green-50 text-green-700' : 'bg-gray-50'}`}>
+          <User className={scannedUser ? 'text-green-500' : 'text-gray-400'} />
+          <span className="font-medium">
+            {scannedUser ? `User ID: ${scannedUser}` : 'Awaiting User QR Code...'}
+          </span>
+          {scannedUser && <CheckCircle2 className="ml-auto text-green-500" />}
+        </div>
+      </div>
+
+      {scannedAsset && scannedUser && (
+        <button 
```

ESLint

[error] Delete `·`

(prettier/prettier)